samples/runtime/reversible-simulator/README.md

```diff
@@ -0,0 +1,25 @@
+# A simple reversible simulator
+
+This samples shows how to use the `IQuantumProcessor` interface to build a
+simple reversible simulator.  A reversible simulator can simulate quantum
```

Comment:
Would it help to note that this is very similar to the existing ToffoliSimulator, but presented as an example of how to use the `IQuantumProcessor` interface?

Reply:
Added note on Toffoli Simulator with link to documentation.